### PR TITLE
feat: Add icon and iconMimetype in shortcuts

### PR DESCRIPTION
**Pull request by KillianCourvoisier**
Thanks for contributing to this documentation. If you added a doctype, please be sure that it is present 

- [ ] in the `Readme.md` index page
- [ ] in the `toc.yml` page
- [ ] https://github.com/cozy/cozy-store/blob/master/src/locales/en.json
- [ ] https://github.com/cozy/cozy-store/blob/master/src/config/permissionsIcons.json
- [ ] https://github.com/cozy/cozy-store/tree/master/src/assets/icons/permissions
- [ ] https://github.com/cozy/cozy-stack/blob/master/assets/locales/en.po
- [ ] https://github.com/cozy/cozy-stack/blob/master/assets/styles/cirrus.css

### CODE REVIEW

File: docs/io.cozy.files.md [hunk 1]
```diff
@@ -213,7 +213,8 @@ They can have `metadata`:
 - `target.mime` is the mime-type of the destination of the link (when it is a
   file)
 - `target.app` is the slug of the destination app (internal links only).
-- `icon` {string?} contains the svg icon.   
+- `icon` {string?} contains the svg icon base64 encoded.
```
Comment (cballevre) [inline docs/io.cozy.files.md:216]: The icon can be also a binary of an svg icon so maybe this description is more generic : `contains svg icon or base64 encoded image`

**Review by cballevre — APPROVED** [review]

File: docs/io.cozy.files.md (hunk 1 above)
Comment (Crash--) [inline docs/io.cozy.files.md:216]: This is not 100% true, we can have directly the binary here (and then we have to encode it to base64) or we can have the base64.

See https://github.com/cozy/cozy-home/pull/1919/files#r1156901276

**Comment by KillianCourvoisier** [comment]
@cballevre @Crash-- , is it better now ?

File: docs/io.cozy.files.md [hunk 2]
```diff
@@ -239,7 +240,9 @@ They can have `metadata`:
       },
       "_type": "io.cozy.files",
       "mime": "image/jpg"
-    }
+    },
+    "icon": "[encoded base64 string of the content of the icon or svg binary]",
```
[the remark this answers is not in the document]
Comment (KillianCourvoisier) [reply docs/io.cozy.files.md:244]: Crash has been too fast ahah

**Pull request merged**